node/code-snippets/README.md

```diff
@@ -0,0 +1,11 @@
+# HubSpot refresh token JavaScripr script
```

Comment:
If we add to code-snippets folder general README, seems its description should be generalized too, so we're able to add more snippets here in the future if necessary. For example title
"Hubspot code snippets scripts for using them *somewhere*"
and after that subtitle about refresh token script. Works also for python.
Also, please, fix the typo to **Javascript** not Javascripr